code/obj/item/material.dm

```diff
@@ -486,6 +486,18 @@
 	default_material = "gold"
 	dense = 2
 
+/obj/item/raw_material/plutonium
+	name = "plutonium ore"
+	desc = "A rock containing just enough radioactive plutonium that it can be smelted."
+	material_name = "Plutonium"
+	default_material = "plutonium"
+
+/obj/item/raw_material/neutronium
+	name = "neutronium ore"
+	desc = "An ore containing deadly neutronium metal."
+	material_name = "Neutronium"
+	default_material = "neutronium"
```

Comment:
This doesn't really make sense for synthetic metals like plutonium, there's no such thing as "plutonium ore". Since they're dropping from nanites now maybe they could be chunks of scrap instead?